zkevm-circuits/src/super_circuit.rs

```diff
@@ -74,6 +75,7 @@ pub struct SuperCircuitConfig<F: Field, const MAX_TXS: usize, const MAX_CALLDATA
     keccak_table: KeccakTable,
     copy_table: CopyTable,
     evm_circuit: EvmCircuit<F>,
+    state_circuit: StateCircuitConfig<F>,
```

Comment:
not sure if need add `state_circuit` into L86 like `tx_circuit` I see it will be used for unit tests

Reply:
nonono `TxCircuit` is witness, while `StateCircuitConfig` is columns....  See for more details: https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/650